src/RequestInspector.tsx

```diff
@@ -79,9 +80,10 @@ function ProofDisplay({ proof } : { proof: Proof }) {
       </TableDisplay>
     )
   } else {
-    return (
-      <pre>{JSON.stringify(proof, null, 2)}</pre>
-    )
+    const index: Record<string, ReactNode> = {
+      'Root CID': proof.toString(),
```

Comment:
question: shouldn't this be `proof.cid.toString()` like above? probably also a good idea to use `<ShortenAndScroll>` here too:

```suggestion
      'Root CID': <ShortenAndScroll>proof.cid.toString()</ShortenAndScroll>,
```

Please test this as I have not verified it works as written.